### DISCUSSION TITLE
[Question] Plugin recreation informant

### DISCUSSION THREAD

**Discussion by ExplosionHm** [aieuo/Mineflow#244] [General]
### Question

Hello, **Aieuo**!

Recently, myself and my friend have been working on a recreation of your plugin for Vanilla Bedrock Edition. Before we continue, we would like to contact you for your permission and/or involvement in the project.

If that intrigues your interest and you would like to discuss this further, please contact me at `explosionhm` on Discord.

_(Sorry for this method of contact; I did not find any contact information linked to your profile.)_

### Plugin Version

undefined

**Comment by aieuo** [comment]
Hello,

Thank you for using Mineflow and for reaching out.

I appreciate your project on Vanilla Bedrock Edition. Feel free to use our code under the NCSA license.

I was actually working on making Mineflow compatible with BDSX as well. If you are using TypeScript, I can provide you with the code (please note that only some parts have been converted, and it is not yet optimized for TypeScript/JavaScript).


---
My discord user id is `aieuo`, icon of the account is same as github.

**Comment by lightyisreal** [comment]
Hello,

Me and my friend have decided to stop development on said plugin recreation due to loss of interest and lack of availability. We're sorry for getting your hopes up. We hope you have a great day :D
